modules/core/src/main/java/org/apache/ignite/internal/visor/verify/VisorValidateIndexesTask.java

```diff
@@ -106,19 +108,19 @@ private static class VisorValidateIndexesJob extends VisorJob<VisorValidateIndex
          * @param arg Argument.
          * @param debug Debug.
          */
-        protected VisorValidateIndexesJob(@Nullable VisorValidateIndexesTaskArg arg, boolean debug) {
+        protected VisorValidateIndexesJob(@Nullable CacheValidateIndexesCommandArg arg, boolean debug) {
             super(arg, debug);
         }
 
         /** {@inheritDoc} */
-        @Override protected VisorValidateIndexesJobResult run(@Nullable VisorValidateIndexesTaskArg arg) throws IgniteException {
+        @Override protected VisorValidateIndexesJobResult run(@Nullable CacheValidateIndexesCommandArg arg) throws IgniteException {
```

Comment:
arg is nullable, but there is no code to cover this case

Reply:
assert added.